storybook/src/docs/icon-gallery.docs.mdx

```diff
@@ -1,7 +1,9 @@
-import { Meta } from "@storybook/blocks";
+import { Meta } from "@storybook/addon-docs/blocks";
 import { AmsterdamIconGallery } from "./components/AmsterdamIconGallery";
 
-<Meta title="Docs/Assets/Icons" />
+{/* Below "parameters" prop does not seem to work; see docs: https://github.com/etchteam/storybook-addon-status?tab=readme-ov-file#story-usage */}
```

Comment:
If this plugin doesn't work here, and we don't want to show the statusses in the TOC anyway, isn't it easier to just make a custom element and place it in the MDX? The currect badges aren't very easy to see imo

Reply:
I updated it to show how it looks with our own Badge component to show the status, but as you can see we then need to deal with the margin above the heading.

The [Badge addon has way more style properties](https://github.com/geometricpanda/storybook-addon-badges?tab=readme-ov-file#preconfigured-badges) which is nice, but I also could not get it to work with the docs. Maybe there is a solution, but I can't figure one out.

Reply:
I'm experimenting with our Badge component again. It's now added below the story's heading and absolutely positioned above it. I did that for the Icon Gallery and the Header. What do you think?

Also @VincentSmedinga

Reply:
Yes, I think it’s fair to say that the conclusion of this experiment is that we can best implement our own status badge. We want it to appear a bit differently than what they provide anyway, and the quality isn’t up to par. Thanks, still :)